### PR TITLE
Add sebastian ictp story

### PR DESCRIPTION
**Pull request by SarahAlidoost**
NB: the title and affiliation are: 

- The representation and identification of three historic "Heavy Precipitation Events" in convection-permitting observations and models 
- The Abdus Salam International Centre for Theoretical Physics - Earth System Physics Section (ICTP-ESP)

However, these are too long for the panel on the home page. In this PR, they are summarized, see the `name` and `description` in index.yml.

### CODE REVIEW

File: content/legacy/6_gard.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+image: legacy/gard.png
+headline: Gard, France
+---
+
+## Gard, France, in September 2002
+
+
+This event is documented in detail by Delrieu et al., 2005. A landfalling mesoscale convective system remained stationary for almost two days, trapped by the orography. It brough horrendous amounts of rainfall and caused wide-spread flooding.
```
Comment (Peter9192) [inline content/legacy/6_gard.md]: Can we make the references link to the corresponding doi's?

File: content/legacy/3_representation.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+image: legacy/representation.png
+headline: Representation of precipitation extremes
+---
+
+## Representation of precipitation extremes
+
+Regional Climate Models integrated on convection permitting grids (dx<=4km; cpRCMs) improve the representation of precipitation extremes, as shown by Pichelli et al. (2021), e.g. in their Figure 3. However, information about the properties of the events, causing these extremes, is not yielded.
```
Comment (Peter9192) [inline content/legacy/3_representation.md]: Can we make the references link to the corresponding doi's?

**Review by Peter9192 — APPROVED** [review]
Really nice storyboard this is as well. Just a question about the links

File: content/legacy/6_gard.md (hunk 1 above)
Comment (SarahAlidoost) [reply content/legacy/6_gard.md]: Done!

File: content/legacy/3_representation.md (hunk 2 above)
Comment (SarahAlidoost) [reply content/legacy/3_representation.md]: Done!

**Pull request merged**